composer.json

```diff
@@ -39,7 +39,7 @@
         "irc": "irc://irc.freenode.org/muffin"
     },
     "require": {
-        "cakephp/cakephp": "^3.6"
```

Comment:
You need to add back cakephp/cakephp under "require-dev" for the test suit to run.

Reply:
Okay. Wouldn't it be better to make unit tests also to use only cakephp/orm?

Reply:
It would need extra effort to create test cases which directly use phpunit's `TestCase` class instead of `Cake\TestSuite\TestCase`. That's something I am not interested in spending time and effort on.